### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by lshanmug**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (niraj-modi) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: A clear and concise description of what the problem is. Ex. I suggest an improvement/enhancement to SWT [...]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[ ] Windows
+[ ] Linux
+[ ] macOS
+ 
+ - Additional OS info (e.g. OS version, Linux Desktop, etc)
+ 
+**Version**
+ Eclipse or SWT version on which the behavior is seen [e.g. 4.23]
+
+**Additional context**
```
Comment (niraj-modi) [inline .github/ISSUE_TEMPLATE/bug_report.md:35]: **Possible workaround or any additional context**

**Review by niraj-modi — CHANGES_REQUESTED** [review]
Looks good overall: suggested some improvements.

**Review by niraj-modi — APPROVED** [review]
Looks good now.

**Comment by lshanmug** [comment]
Closed as ECA was failing, new PR created with correct email-id: #178

**Pull request closed without merging**